### PR TITLE
Support GCP

### PR DESCRIPTION
**Pull request by x-wf**
**New commands**
- `import` - import files for into the _.radnet_ folder.  Currently has the `-a` argument is to provide the path for the authentication file. (e.g. `radnet import -a ~/Desktop/platforms.json`). This brings the simplicty of not having to go through the hastle of copying the file to _.radnet_ folder manually.

**Changes**
- `authentication`: New `platforms.json` file in 1Password, supporting **GCP** (yay!).
- `init`: made GCP default platform.
- `init`: added `-z`,`-p` flags.

**Todo:**
- Integrate with current Resource Cleaner

This is for an early code review.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,17 +24,16 @@ To install *radnet*, you need to download it with *pip*. After installing *radne
 
 ### Platforms
 To spin up networks, you must have proper authentication to the target platform. 
-The credentials need to be placed under `.radnet` folder.
+The credentials need to be placed imported.
```
Comment (blenessy) [inline README.md]: "placed" or "imported" to where ?

File: client/radnet/util/network.py [hunk 2]
```diff
@@ -8,13 +8,17 @@
...
 import getpass
 import time
+import json
 from radnet.util import util
 from radnet.util import node
 from radnet.util.exceptions import TerraformProvidersException
 
-PLATFORM_AZURE = 1
-PLATFORM_GCP = 2
+PLATFORM_AZURE = "az"
```
Comment (blenessy) [inline client/radnet/util/network.py]: The `gcp` abbreviation makes sense however, can we do `azure` its not much longer than `az` ?

(Note you need to change the doc too)

File: client/setup.py [hunk 3]
```diff
@@ -78,7 +78,8 @@ def check_terraform_installation():
    include_package_data=True,
    install_requires=[
       'Jinja2==2.10',
-      'paramiko'
+      'paramiko',
+      'scp',
```
Comment (blenessy) [inline client/setup.py]: Can we revert back to `sftp` now that we know that it was not the culprit ?

File: README.md (hunk 1 above)
Comment (x-wf) [reply README.md]: Into the _radnet_ folder. 😄 
Fixed typo

**Comment by x-wf** [comment]
@blenessy it is now ready for a last review, merge when happy 👍

**Pull request merged**